letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -152,10 +152,11 @@ su_sudo() {
   su root -c "$args"
 }
 
-if [ "$1" = "--cb-auto-has-root" ]; then
-  shift 1
-elif [ "$1" != "--le-auto-phase2" ]; then
-  # if $1 is --le-auto-phase2, we've executed this branch before
+# Sets the environment variable SUDO to be the name of the program or function
+# to call to get root access. If this script already has root privleges, SUDO
+# is set to an empty string. The value in SUDO should be run with the command
+# to called with root privileges as arguments.
+set_root_auth_mechanism() {
```

Comment:
Maybe we could standardize the subroutine name casing conventions while we're in here. This one arguably introduces a new convention. Most seem to be UpperCamelCase, with su_sudo as an outlier and some single-word lowercase ones thrown in for good measure.